examples/historic_async.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import asyncio
+import alpaca_trade_api as tradeapi
+from alpaca_trade_api.rest import TimeFrame, URL
+from alpaca_trade_api import rest_async
+from alpaca_trade_api.rest_async import gather_with_concurrency
+from alpaca_trade_api import AsyncRest
+import pandas as pd
+
+NY = 'America/New_York'
+
+
+async def get_historic_bars(symbols, start, end, timeframe: TimeFrame):
+    major = sys.version_info.major
+    minor = sys.version_info.minor
+    if major < 3 or minor < 6:
+        raise Exception('asyncio is not support in your python version')
+    print(f"Getting data for {len(symbols)} symbols, timeframe: {timeframe},"
+          f" between dates: start={start}, end={end}")
+
+    tasks = []
+
+    for symbol in symbols:
+        tasks.append(
+            rest.get_bars_async(symbol, start, end, timeframe, limit=500))
+    if minor >= 8:
+        results = await asyncio.gather(*tasks)
+    else:
+        results = await gather_with_concurrency(500, *tasks)
+
+    bad_requests = 0
+    for response in results:
```

Comment:
It's not trivial how to actually consume the responses (other than printing their count). E.g. how to pair the symbol with the dataframe.

Reply:
now returning the symbol with the dataframe

Reply:
I'm still missing to see it in an example.

Reply:
now each function returns a tuple including the symbol

![image](https://user-images.githubusercontent.com/78740792/132317647-d16baf18-652d-4aa4-a791-87f9ed50eb5a.png)